spec/system/referee_interface/referee_cannot_provide_reference_if_two_already_received_spec.rb

```diff
@@ -0,0 +1,64 @@
+require 'rails_helper'
+
+RSpec.feature 'Referee can submit reference', with_audited: true do
```

Comment:
Is auditing needed for this spec?

Reply:
No, think this was a copy / paste error. I'll update the description as well.